R/endpoints.R

```diff
@@ -1,9 +1,9 @@
 api_build <- function(queue) {
   pr <- plumber::plumber$new()
-  pr$handle("POST", "/validate/input", endpoint_validate_input,
-            serializer = serializer_json_hintr())
   pr$handle("POST", "/validate/baseline", endpoint_validate_baseline,
             serializer = serializer_json_hintr())
+  pr$handle("POST", "/validate/baseline-all", endpoint_validate_baseline_all,
```

Comment:
baseline vs baseline-all don't seem terribly intuitive to me, I'm afraid. `baseline-individual`, `baseline-combined`?

Reply:
Good yeah, that better shows the intent. I've updated that.